index.js

```diff
@@ -20,7 +20,7 @@ function isFunction(f) {
 
 module.exports = {
 
-  browser: process.browser,
+  browser: typeof process === "undefined" || !process.nextTick || Boolean(process.browser),
```

Comment:
It's less clear though, and this check only happens once so I don't see a need to optimize it for speed.  It's the same reason I thought that your version of `isCipherAvailable` that uses `.some` is preferable to the faster (but clunkier) version that just used a loop!